public/index.html

```diff
@@ -16,7 +17,9 @@
     <title>Flashcards</title>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="background-image">
```

Comment:
Why do we need a `#background-image` element if you are applying the background image to `body`?